cves/CVE-2011-1184.yml

```diff
@@ -43,41 +43,40 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+    Users can look at traffic on a line, and get autheticated information. Then they
+    can use it by repeatedly sending the server the information. In this case the server
+    will give the attacker autheticated privlidges. 
 
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
 bounty:
   amt:
-  announced: 2012-01-14
+  announced:
   url:
+  
 bugs: []
+
 fixes_vcc_instructions: |
   Please put the Git commit SHA in "commit" below, and any notes about how this
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: 644dfdf96cf82fcd2a2046d93f2b5495f7e94584
-     note: SVN rev 1159309, from the Tomcat website.
-   - commit:
-     note:
+   - commit: 639e20992a66d7a42fb59c974db91c8a0f730a1e
+     note: Added a function to check the nonce. The nonce is used to check if the message is old or already used. Now attackers can not use the same authentication message they found. Since the server will be looking for a different nonce value. 
 vccs:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: b7b5c63a932f6c1ea05f9b65ad9054247bb5af57
+    note: They added preemptive authentication, but never added checks to if the authetication was already done. Missed a corner case / did not know about the corner case. 
 
 incomplete_fix_instructions: |
   Did the above "fixes" actually fix the vulnerability?
   Please list any fix commits for this vulnerability that had to be corrected
   at a later date.
 incomplete_fixes:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 639e20992a66d7a42fb59c974db91c8a0f730a1e
```

Comment:
This seems to be a completed fix, so I don't think anything needs to be added to the `incomplete_fixes` section

Reply:
Yup misread that section!